channeld/full_channel.c

```diff
@@ -562,7 +562,7 @@ static bool local_opener_has_fee_headroom(const struct channel *channel,
 
 	/* Now, how much would it cost us if feerate increases 100% and we added
 	 * another HTLC? */
-	fee = commit_tx_base_fee(2 * feerate, untrimmed + 1,
+	fee = commit_tx_base_fee(1.1 * feerate, untrimmed + 1,
```

Comment:
maybe the answer is inside the issue description, but would be good to add a comment on why we are adding this const value?

Reply:
Added a comment in the commit description.